src/plugins/orangehrmClaimPlugin/Api/ClaimExpenseTypeAPI.php

```diff
@@ -41,25 +46,100 @@ class ClaimExpenseTypeAPI extends Endpoint implements CrudEndpoint
 
     public const PARAMETER_NAME = 'name';
     public const PARAMETER_DESCRIPTION = 'description';
-
+    public const PARAMETER_ID = 'id';
+    public const PARAMETER_IDS = 'ids';
+    public const PARAMETER_EXPENSEID = 'expenseId';
     public const PARAMETER_STATUS = 'status';
     public const DESCRIPTION_MAX_LENGTH = 1000;
     public const NAME_MAX_LENGTH = 100;
 
     /**
+     * @OA\Get(
+     *     path="/api/v2/claim/expenses/types",
+     *     tags={"Claim/ExpenseTypes"},
+     *     @OA\Parameter(
+     *         name="sortField",
+     *         in="query",
+     *         required=false,
+     *         @OA\Schema(type="string", enum=ClaimExpenseTypeSearchFilterParams::ALLOWED_SORT_FIELDS)
+     *     ),
+     *     @OA\Parameter(ref="#/components/parameters/sortOrder"),
+     *     @OA\Parameter(ref="#/components/parameters/limit"),
+     *     @OA\Parameter(ref="#/components/parameters/offset"),
+     *     @OA\Response(
+     *         response="200",
+     *         description="Success",
+     *         @OA\JsonContent(
+     *             @OA\Property(
+     *                 property="data",
+     *                 type="array",
+     *                 @OA\Items(ref="#/components/schemas/Claim-ClaimExpenseTypeModel")
+     *             ),
+     *             @OA\Property(property="meta",
+     *                 type="object",
+     *                 @OA\Property(property="total", type="integer")
+     *             )
+     *         )
+     *     )
+     * )
      * @inheritDoc
      */
     public function getAll(): EndpointResult
     {
-        throw $this->getNotImplementedException();
+        $claimExpenseTypeSearchFilterParams = new ClaimExpenseTypeSearchFilterParams();
+        $this->setSortingAndPaginationParams($claimExpenseTypeSearchFilterParams);
+        $claimExpenseTypeSearchFilterParams->setName(
+            $this->getRequestParams()->getStringOrNull(RequestParams::PARAM_TYPE_QUERY, self::PARAMETER_NAME)
+        );
+        $claimExpenseTypeSearchFilterParams->setStatus(
+            $this->getRequestParams()->getBooleanOrNull(RequestParams::PARAM_TYPE_QUERY, self::PARAMETER_STATUS)
+        );
+        $claimExpenseTypeSearchFilterParams->setId(
+            $this->getRequestParams()->getIntOrNull(RequestParams::PARAM_TYPE_QUERY, self::PARAMETER_EXPENSEID)
```

Comment:
$this->getRequestParams()->getIntOrNull(RequestParams::PARAM_TYPE_QUERY, self::PARAMETER_EXPENSE_TYPE_ID)